### DISCUSSION TITLE
Generalize the article reaction images

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16201] [Feature Requests]
I feel that there is a case to generalise the reaction images for the 3 actions you can take on an article (heart, unicorn, save). I think it helps with a bit of uniqueness throughout the Forem ecosystem as it grows.

**Comment by Andres7132** [comment]
How will we apply these features so that we could detect the reactions for https://prosawtool.com/best-cordless-miter-saws/ . As we are measuring it manually but want to apply some automation so that it could make our work easy. As I have to make my business automated. If you have any idea then kindly share I will appreciate your suggestions.
